app/components/instructor-group/courses.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+import { use } from 'ember-could-get-used-to-this';
+import ResolveAsyncValue from 'ilios-common/classes/resolve-async-value';
+
+export default class InstructorGroupCoursesComponent extends Component {
+  @use courses = new ResolveAsyncValue(() => [this.args.instructorGroup.courses, []]);
```

Comment:
This file seems unneeded as `instructorGroup.courses` isn't a promise, it's a getter on the model that already returns an empty array before it is resolved.

Reply:
:heavy_check_mark: